### ISSUE TITLE
System unresponsive after turn off Accessibility permission [bug]

### ISSUE THREAD

**Issue by dingwen07** [noah-nuebling/Mac-Mouse-Fix#443]
## Description

If I turn off Accessibility permission while MMF is running, then the entire computer will stop responding IO (Keyboard, Touchpad, Mouse). I have to SSH into the computer and run `killall "Mac Mouse Fix Helper"`.

## Steps to reproduce

0. Make sure SSH is enabled and you can login with your current account
1. Open MMF
2. Grant Accessibility permission in Settings
3. Enable MMF
4. Revoke Accessibility permission in Settings
5. Observe that System stop responding
6. SSH into the computer and run `killall "Mac Mouse Fix Helper"`

## Context

Mac Mouse Fix Version: 3.0.0 Beta 5
macOS Version: macOS 13.0.1 (22A400)
Mouse Model: N/A

## Additional Comments

–

---

### Console Logs

–

### Crash Reports

–

**Comment by noah-nuebling** [comment]
Hi @dingwen07, thanks for reaching out!

As far as I know, this is a macOS problem. Any app that catches input events will have the same issue. If anybody knows a solution or an app where this is not the case, let me know!

Thank you.

**Comment by dgatwood** [comment]
Ran into this bug, too.  The only solution was a hard reboot.  I don't see this when I remove the accessibility checkbox from, for example, VLC (even when VLC is running), so I can only assume that it is a side effect of the way that you're capturing events and no longer passing them on.

Are there error codes when delivering an event fails?  It should presumably be possible to catch those errors and remove the capture hooks.  That might fix the problem.  (While it was in this state, the system was responsive, but getting slower and slower, and mouse motion events worked, but clicks were lost.)
